app/uk/gov/hmrc/tariffclassificationfrontend/views/cancel_ruling.scala.html

```diff
@@ -0,0 +1,58 @@
+@*
+ * Copyright 2019 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *@
+
+@import uk.gov.hmrc.tariffclassificationfrontend.config.AppConfig
+@import uk.gov.hmrc.tariffclassificationfrontend.forms.ReleaseCaseForm
+@import uk.gov.hmrc.tariffclassificationfrontend.models.{BTIApplication, Case, Queue}
+@import uk.gov.hmrc.tariffclassificationfrontend.views.html.forms.release_case_form
+@import uk.gov.hmrc.tariffclassificationfrontend.views.html.includes.main
+@import uk.gov.hmrc.tariffclassificationfrontend.views.html.partials.case_heading
+@import views.html.helper.{CSRF, form}
+
+@(c: Case)(implicit request: Request[_], messages: Messages, appConfig: AppConfig)
+
+@main(bodyClasses = None) {
+
+    @case_heading(c)
+
+    <div class="grid-row">
```

Comment:
I'd maybe add an ID to the top level container to help with acceptance tests

Reply:
done